README.md

```diff
@@ -6,27 +6,78 @@
 
 # Quantum serverless
 
+Quantum Serverless is a user-friendly tool that enables you to easily run complex quantum computing tasks. 
+With this software, you can execute Qiskit programs as long running jobs and distribute them across multiple CPUs, GPUs, and QPUs. 
+This means you can take on more complex quantum-classical programs and run them with ease. 
+You don't have to worry about configuration or scaling up computational resources, as Quantum Serverless takes care of everything for you. 
+
 ![diagram](./docs/images/qs_diagram.png)
 
 ### Table of Contents
 
 1. [Installation](INSTALL.md)
 2. [Quickstart](#quickstart-guide)
 3. [Beginners Guide](docs/beginners_guide.md)
-4. Modules:
+4. [Getting started](docs/getting_started/)
+5. Modules:
    1. [Client](./client)
    2. [Infrastructure](./infrastructure)
-5. [Tutorials](docs/tutorials/)
-6. [Guides](docs/guides/)
-7. [How to Give Feedback](#how-to-give-feedback)
-8. [Contribution Guidelines](#contribution-guidelines)
-9. [References and Acknowledgements](#references-and-acknowledgements)
-10. [License](#license)
+6. [Tutorials](docs/tutorials/)
+7. [Guides](docs/guides/)
+8. [How to Give Feedback](#how-to-give-feedback)
+9. [Contribution Guidelines](#contribution-guidelines)
+10. [References and Acknowledgements](#references-and-acknowledgements)
+11. [License](#license)
 
 ----------------------------------------------------------------------------------------------------
 
 ### Quickstart
 
+
+
+<details>
+  <summary>1. Hello World quickstart</summary>
+
+1 - Create program file `hello_qiskit.py`
+```python
+# hello_qiskit.py
+
+from qiskit import QuantumCircuit
+from qiskit.primitives import Sampler
+
+circuit = QuantumCircuit(2)
+circuit.h(0)
+circuit.cx(0, 1)
+circuit.measure_all()
+circuit.draw()
+
+sampler = Sampler()
+
+quasi_dists = sampler.run(circuit).result().quasi_dists
+
+print(f"Quasi distribution: {quasi_dists[0]}")
+```
+
+2 - Run program file
+```python
+from quantum_serverless import QuantumServerless, Program
+serverless = QuantumServerless(...) # serverless setup is provided by your admin or use docker compose (refer to all-in-one quickstart)
```

Comment:
the all-in-one is step 2, so I'd add the docker-compose setup before this (so that users don't have to stop and go back to setup their environment)

Reply:
I simplified it to bare minimum. Jen will conduct user research to figure out what is the best way to describe getting started.